CHANGELOG.md

```diff
@@ -4,6 +4,15 @@
 
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 
+## 7.20.0
+
+- feat(angular): Add Angular 15 Peer Dependencies (#6220)
+- feat(nextjs): Add `excludeServerRoutes` config option (#6207)
+- feat(node): Move tracing options to `Http` integration (#6191)
+- fix(remix): Resolve Remix Request API compatibility issues. (#6215)
+- fix(nextjs): Use absolute path for `distDir` in webpack plugin options (#6214)
```

Comment:
```suggestion
- fix(nextjs): Use absolute path for `distDir` in webpack plugin options (#6214)
- fix(remix): Resolve Remix Request API compatibility issues. (#6215)
```

Reply:
God every time - this time I actually tried to do it right lol